ShowdownAI.Middleware/ShowdownAI.Middleware/Models/Pokemon.cs

```diff
@@ -2,20 +2,31 @@
 {
     public class Pokemon
     {
-        public string Ident { get; set; }
-        public string Details { get; set; }
-        public string Condition { get; set; }
+        public string Ability { get; set; }
```

Comment:
Didn't look like there was any particular ordering to the properties, I'm a fan of alphabetical sorting in. Would have made checking for this property in the first place a bit quicker for me.

Reply:
Practically it doesn't matter the order. Normally I try to keep it matching the JSON structure but shouldn't matter too much so I'll approve that.